### ISSUE TITLE
Add icon support

### ISSUE THREAD

**Issue by fsmaia** [sgwozdz/jenkinsfile-support#26]
I just added our `jenkinsfile` alias to `vscode-icons` repository:
https://github.com/vscode-icons/vscode-icons/pull/2116

I'm starting this issue just for discussion if changes are required in this repository (e.g. renaming the alias to `jenkins`).

Hope it gets merged soon :tada:

**Comment by britho** [comment]
I'm curious about what would happen if we changed the alias to "jenkins" in this repo and someone was using an icon theme other than vscode-icons, like Monokai Pro for instance.  Would their Jenkinsfile support just go away?  I'm sure we could add backward compatibility if that's the case.

**Comment by fsmaia** [comment]
Hmm :thinking: 
What's the most straightforward alias for Jenkinsfile? Just couldn't find out Monokai Pro icon names. If it is `jenkinsfile` so we could keep our current name, as we expect a merge in `vscode-icons`.

**Comment by britho** [comment]
In hindsight, I think `jenkins` is more than sufficient.  I'm not sure if any code changes here are necessary.  Perhaps someone else could shed some light on this.

**Comment by sgwozdz** [comment]
TL;DR
If you want to have icons for your jenkins files, you have to add it to the icon theme you're using.

> I'm curious about what would happen if we changed the alias to "jenkins" in this repo and someone was using an icon theme other than vscode-icons, like Monokai Pro for instance. Would their Jenkinsfile support just go away? I'm sure we could add backward compatibility if that's the case.

Icon themes and extensions to support languages are completely separate things. But both of them decide when to apply its logic based on the same rule - file name/extension. So in theory, if you add something here:
```
 "extensions": [
     "Jenkinsfile",
     "jenkinsfile",
     ".jdp"
 ],
 "filenames": [
     "Jenkinsfile",
     "jenkinsfile"
 ]
```

User will see this extension features in his file with the same extension or name.
I've added similar to the theme I'm using - https://github.com/PKief/vscode-material-icon-theme/pull/502

**Comment by sgwozdz** [comment]
@fsmaia your change to vscode-icons seems to deployed, can we close the ticket?

**Comment by fsmaia** [comment]
For sure! Big thanks :)
